sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/math.scala

```diff
@@ -255,3 +257,39 @@ case class Pow(left: Expression, right: Expression)
       """
   }
 }
+
+case class Round(left: Expression, right: Expression)
```

Comment:
left and right are here for implement the abstract fields of the super class `BinaryNode`, so I guess we cann't refactor the name here at will